cmd/webhook.go

```diff
@@ -29,9 +29,7 @@ var (
 
 const (
 	admissionWebhookAnnotationInjectKey = "kerberos-injector-webhook.das-zone.statcan/inject"
```

Comment:
I do want to mention, intuitively the names for these is confusing. I expect `inject` to be bool `true/false` but that doesn't make sense here, and `status` to be `injected` or ` ` if not. Something like that. But it seems that `inject` = `injected` is done here.